cpp/daal/src/algorithms/kernel_inst_arm.h

```diff
@@ -0,0 +1,131 @@
+/* file: kernel_inst_arm.h */
```

Comment:
@Vika-F that PR does not show any SYCL related modifications, could you please help point it out?

Reply:
@rakshithgb-fujitsu  - this is not part of this PR, but a comment for longerterm plans.  all sycl related finctionality in DAAL library is deprecated and will be removed.  so there is no point in copying DAAL SYCL related defines.

Reply:
@rakshithgb-fujitsu Yes, my point was about the deprecation of SYCL-related functionality in DAAL. Since it is deprecated, no need to replicate those defines for ARM. They will be removed in near future anyway.

Now I also see that it is not easy to remove those defines at all right now as they are used in various files.

So, let's leave them as is for now in order not to block the development anymore.